### PR TITLE
Add Editor.js plugin

### PR DESCRIPTION
**Pull request by melishev**
#### By submitting this pull request, I promise that:

- [x] I have read the [contribution guidelines](https://github.com/strapi/awesome-strapi/blob/master/CONTRIBUTING.md) thoroughly.
- [x] I ensure my submission follows each and every point.

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
Hey @melishev thanks for the PR, I had some trouble testing this on our latest version (v3.6.6).

Full log (on a fresh project) is below. Steps to reproduce issue:

- create a new project: `npx create-strapi-app@latest testProject --quickstart --no-run`
- install plugin: `yarn add strapi-plugin-react-editorjs`
- attempt to start project (including building the admin)
- See error.

At this time I can't merge this PR until the issues with the plugin are resolved.

```
dmehaffy@DM-Desktop-LM:/mnt/storage/Development/testEditorjs$ yarn add strapi-plugin-react-editorjs
yarn add v1.22.10
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.3.2: The platform "linux" is incompatible with this module.
info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning "strapi-admin > bootstrap@4.6.0" has unmet peer dependency "jquery@1.9.1 - 3".
warning "strapi-admin > bootstrap@4.6.0" has unmet peer dependency "popper.js@^1.16.1".
warning "strapi-plugin-users-permissions > grant-koa@5.4.8" has unmet peer dependency "koa@>=2.0.0".
[5/5] Building fresh packages...

success Saved lockfile.
success Saved 36 new dependencies.
info Direct dependencies
└─ strapi-plugin-react-editorjs@1.3.4
info All dependencies
├─ @editorjs/checklist@1.3.0
├─ @editorjs/code@2.7.0
├─ @editorjs/delimiter@1.2.0
├─ @editorjs/editorjs@2.22.2
├─ @editorjs/embed@2.4.6
├─ @editorjs/header@2.6.1
├─ @editorjs/image@2.6.0
├─ @editorjs/inline-code@1.3.1
├─ @editorjs/link@2.3.1
├─ @editorjs/list@1.6.2
├─ @editorjs/marker@1.2.2
├─ @editorjs/paragraph@2.8.0
├─ @editorjs/quote@2.4.0
├─ @editorjs/raw@2.3.0
├─ @editorjs/table@2.0.1
├─ @editorjs/warning@1.2.0
├─ @types/cacheable-request@6.0.2
├─ @types/http-cache-semantics@4.0.1
├─ @types/keyv@3.1.2
├─ @types/responselike@1.0.0
├─ cacheable-lookup@5.0.4
├─ chardet@1.3.0
├─ cheerio-select@1.5.0
├─ cheerio@1.0.0-rc.10
├─ codex-notifier@1.1.2
├─ codex-tooltip@1.0.2
├─ get-file-object-from-local-path@1.0.2
├─ got@11.8.2
├─ http2-wrapper@1.0.3
├─ open-graph-scraper@4.9.1
├─ parse5-htmlparser2-tree-adapter@6.0.1
├─ quick-lru@5.1.1
├─ react-editor-js@1.10.0
├─ resolve-alpn@1.2.0
├─ strapi-plugin-react-editorjs@1.3.4
└─ validator@13.6.0
Done in 20.37s.
dmehaffy@DM-Desktop-LM:/mnt/storage/Development/testEditorjs$ yarn develop
yarn run v1.22.10
$ strapi develop
Building your admin UI with development configuration ...

● Webpack █████████████████████████ building (68%) 1485/1523 modules 38 active
 ...ns/strapi-plugin-content-type-builder/admin/src/components/RelationTargetPicker/Wrapp
er.js

Error: Module not found: Error: Can't resolve '../mediaLib/utils' in '/mnt/storage/Development/testEditorjs/.cache/plugins/strapi-plugin-react-editorjs/admin/src/components/editorjs'
    at /mnt/storage/Development/testEditorjs/node_modules/strapi-admin/index.js:71:23
    at finalCallback (/mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compiler.js:257:39)
    at onCompiled (/mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compiler.js:265:20)
    at /mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compiler.js:670:21
    at eval (eval at create (/mnt/storage/Development/testEditorjs/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:28:1)
    at /mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compilation.js:1173:13
    at /mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compilation.js:1096:25
    at /mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compilation.js:1017:13
    at /mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2830:7
    at /mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2830:7
    at done (/mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2863:11)
    at /mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2818:7
    at /mnt/storage/Development/testEditorjs/node_modules/webpack/lib/Compilation.js:1017:13
    at /mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2830:7
    at /mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2830:7
    at done (/mnt/storage/Development/testEditorjs/node_modules/neo-async/async.js:2863:11)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
dmehaffy@DM-Desktop-LM:/mnt/storage/Development/testEditorjs$ 
```

**Comment by melishev** [comment]
Hi, @derrickmehaffy, I just checked your point. I tried to repeat the steps described by you, but I could not stumble upon your mistake. Judging by the logs, I can assume that the problem lies in the name of the folder. I corrected it, please check it. Thanks a lot!

**Review by derrickmehaffy — APPROVED** [review]
LGTM, the install worked after requested changes. :)

**Pull request merged**
